client/src/config/Routes.js

```diff
@@ -1,14 +1,20 @@
 import React from "react";
 import { Route, Switch } from "react-router-dom";
 import Home from "../views/Home";
+import Login from "../views/Login";
+import Signup from "../views/SignUp";
+import Account from "../views/AccountSettings";
 import ResultsTable  from "../components/ResultsTable";
 
 export default class Routes extends React.Component{
   render(){
     return(
       <Switch>
         <Route path="/" exact component={Home} />
-        <Route path="/results" exact render={ () => <ResultsTable/> } />
+        <Route path="/results" exact render={ () => <ResultsTable results={this.props.results}/> } />
```

Comment:
I don't think ResultsTable has a results prop anymore, Cormac updated it a while ago.